src/harness/reference_models/dpa/dpa_mgr.py

```diff
@@ -135,27 +137,40 @@ def __init__(self, protected_points,
                beamwidth=3,
                azimuth_range=(0, 360),
                freq_ranges_mhz=[(3550, 3650)],
-               catb_neighbor_dist=ml.CAT_B_NBRHD_DIST_DEFAULT):
+               catb_neighbor_dist=ml.CAT_B_NBRHD_DIST_DEFAULT,
+               dpa_type='esc'):
     """Initialize the DPA attributes."""
     self.protected_points = protected_points
     self.threshold = threshold
     self.radar_height = radar_height
     self.azimuth_range = azimuth_range
     self.beamwidth = beamwidth
-    self.channels = GetDpaProtectedChannels(freq_ranges_mhz)
     self.catb_neighbor_dist = catb_neighbor_dist
+    self.dpa_type = dpa_type
+    self.channels = None
```

Comment:
Could you please explain why we need to set "self.channels = None" before calling ResetFreqRange()?

Reply:
No real need. It was just to have all the actual attributes clearly showed in the __init__, so a reader can see exactly what are the attributes. It is bad design to spread attributes (especially public ones) all over the place. As a reader of the code I want to check in the __init__ and see exactly what is the state variable of my class.